static/templates/user_sidebar_actions.handlebars

```diff
@@ -18,5 +18,11 @@
       {{#tr this}}Compose a message to <b>__name__</b>{{/tr}}
     </a>
   </li>
+  <li>
+    <a class="star_user">
+      <i class="icon-vector-star"></i>
+      {{#tr this}}{{#if starred}}Unstar{{else}}Star{{/if}} user <b>__name__</b>{{/tr}}
```

Comment:
This didn't seem to work for me. It always said "Star user ____" even if the user was already starred.